src/components/FindPark.vue

```diff
@@ -40,12 +43,12 @@ export default {
   },
   methods: {
     async search() {
-      const results = await getSearch(this.searchTerm)
+      const results = await getSearch(this.searchTerm);
       this.$store.commit('storeResults', results);
       this.searchTerm = '';
     },
     async searchByLocation() {
-      const results = await getResults(this.$store.state.geolocation.coords)
+      const results = await getResults(this.$store.state.geolocation.coords);
```

Comment:
Thanks for keeping me honest on these semicolons 😅

Reply:
Ditto on this.

Reply:
It's not me! It's the same auto-formatter that messed up my test, lol.